### DISCUSSION TITLE
I am getting an error after deploying to vercel

### DISCUSSION THREAD

**Discussion by Luckyhv** [DuCanhGH/next-pwa#141] [Q&A]
Uncaught (in promise) bad-precaching-response: bad-precaching-response :: [{"url":"https://aniplay-next.vercel.app/swe-worker-development.js","status":404}]

TypeError: f.map is not a function

Can anyone tell me what the issue is related to.

**Comment by DuCanhGH** [comment]
@Luckyhv  Not sure what happened to your pipeline, but https://aniplay-next.vercel.app/swe-worker-development.js is expected to be accessible. Do check if the file is in the "public" directory (if you use the default settings, in which `dest` is set to `"public"`) after the app has been built. Thanks!
